README.md

```diff
@@ -158,6 +158,7 @@ export default defineNuxtConfig({
   modules: ['@sidebase/nuxt-auth'],
   auth: {
     // The module is enabled. Change this to disable the module
+    // Note: Optional and true by default, don't include this line.
```

Comment:
please remove this -> this section is for configuration example as stated on top, many of the below keys are optional.